entrypoint.sh

```diff
@@ -74,6 +74,7 @@ function create_ecr_repo() {
       if echo ${output} | grep -q RepositoryNotFoundException; then
         echo "== REPO DOESN'T EXIST, CREATING.."
         aws ecr create-repository --region $AWS_DEFAULT_REGION --repository-name $INPUT_REPO
+        aws ecr set-repository-policy --region $AWS_DEFAULT_REGION --repository-name $INPUT_REPO --policy-text file://"ecr_policy.json"
```

Comment:
where is `ecr_policy.json` coming from?

Reply:
im actually reverting that, but will be in each repo

Reply:
it is another file in the repository at the same filesystem level as the shell script